<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>7</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Arduino UI</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:pressed {
	background: #90C8F6;
	border: 1px solid #ADADAD;
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>–£–ø—Ä–∞–≤–ª–µ–Ω–∏–µ</string>
     </attribute>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>931</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>211</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–£–ø—Ä–∞–≤–ª–µ–Ω–∏–µ RGB Led</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>171</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="Slider_red">
          <property name="cursor">
           <cursorShape>SizeVerCursor</cursorShape>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="Slider_green">
          <property name="cursor">
           <cursorShape>SizeVerCursor</cursorShape>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="Slider_blue">
          <property name="cursor">
           <cursorShape>SizeVerCursor</cursorShape>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>240</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>G</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_red">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_green">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_blue">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>150</y>
        <width>211</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–£–ø—Ä–∞–≤–ª–µ–Ω–∏–µ —Ä–µ–ª–µ</string>
      </property>
      <widget class="QPushButton" name="btnRelay">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>131</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>–í–∫–ª / –≤—ã–∫–ª</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelRelay">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>171</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>üî¥ –í—ã–∫–ª—é—á–µ–Ω–æ</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>331</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–û—Ç–ø—Ä–∞–≤–∫–∞ —Å–æ–æ–±—â–µ–Ω–∏—è</string>
      </property>
      <widget class="QPushButton" name="serialTextBtn">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>–û—Ç–ø—Ä–∞–≤–∏—Ç—å</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serialText">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>371</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–í—ã–±–æ—Ä COM –ø–æ—Ä—Ç–∞</string>
      </property>
      <widget class="QComboBox" name="comPortsCB">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="openBtn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>–û—Ç–∫—Ä—ã—Ç—å</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeBtn">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>–ó–∞–∫—Ä—ã—Ç—å</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reloadBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>31</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#reloadBtn{
	border: none;	
}</string>
       </property>
       <property name="text">
        <string>üîÑ</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>211</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–£–ø—Ä–∞–≤–ª–µ–Ω–∏–µ —Å–≤–µ—Ç–æ–¥–∏–æ–¥–æ–º</string>
      </property>
      <widget class="QLabel" name="labelLed">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>üî¥ –í—ã–∫–ª—é—á–µ–Ω–æ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLed">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>131</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>–í–∫–ª / –≤—ã–∫–ª</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>451</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–£–ø—Ä–∞–≤–ª–µ–Ω–∏–µ (–®–ò–ú)</string>
      </property>
      <widget class="QSlider" name="pwmSlider">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLCDNumber" name="pwmLcd">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>9</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–°–æ—Å—Ç–æ—è–Ω–∏–µ –ø–æ–¥–∫–ª—é—á–µ–Ω–∏—è</string>
      </property>
      <widget class="QLabel" name="connectBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>üî¥ –í—ã–∫–ª—é—á–µ–Ω–æ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_12">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>211</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–£–ø—Ä–∞–≤–ª–µ–Ω–∏–µ –ø–∏—â–∞–ª–∫–æ–π</string>
      </property>
      <widget class="QLabel" name="labelBuzzer">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>üî¥ –í—ã–∫–ª—é—á–µ–Ω–æ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBuzzer">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>131</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>–í–∫–ª / –≤—ã–∫–ª</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>–î–∞—Ç—á–∏–∫–∏</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–°—á–µ—Ç—á–∏–∫ —ç–Ω–∫–æ–¥–µ—Ä–∞</string>
      </property>
      <widget class="QLCDNumber" name="encLcdNumR">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLCDNumber" name="encLcdNumL">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>–ù–∞–ø—Ä–∞–≤–æ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>–ù–∞–ª–µ–≤–æ</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="encLcdNumClick">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>–ö–ª–∏–∫–∏</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="encoderProgressBar">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>391</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–î–∞—Ç—á–∏–∫ –•–æ–ª–ª–∞</string>
      </property>
      <widget class="QProgressBar" name="hallProgressBar">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>–ù–µ—Ç</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>110</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>–ï—Å—Ç—å</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>–ì–æ–ª–æ—Å–æ–≤–æ–µ —É–ø—Ä–∞–≤–ª–µ–Ω–∏–µ</string>
     </attribute>
     <widget class="QPushButton" name="voiceBtn">
      <property name="geometry">
       <rect>
        <x>921</x>
        <y>20</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#voiceBtn {
	font-size: 100px;
    text-align: center;
	border: none;
	border-radius: 10px
}

#voiceBtn:pressed {
	font-size: 110px;
}</string>
      </property>
      <property name="text">
       <string>üéô</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="voiceConsole">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>521</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>–ö–æ–Ω—Å–æ–ª—å –æ—Ç–ª–∞–¥–∫–∏</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>–í–∫–ª—é—á–µ–Ω–∏–µ</string>
      </property>
      <widget class="QSlider" name="voiceSwitch">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>On</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </widget>
     <widget class="QProgressBar" name="voiceIndicator">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>350</y>
        <width>51</width>
        <height>161</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>410</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>üî¥</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>370</y>
        <width>101</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="text">
       <string>üéô</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
